### PR TITLE
Fix errors in light theme

### PR DESCRIPTION
**Pull request by spryor**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Fix some colors in the light theme

### CODE REVIEW

**Comment by asdonald** [comment]
@hegate  as FYI

File: src/components/shared/forms/styles/select.scss [hunk 1]
```diff
@@ -342,21 +348,22 @@ $select-padding-horizontal: 8px;
   	@include rem-fallback(padding-top, $select-padding-vertical);
   	@include rem-fallback(padding-bottom, $select-padding-vertical);
   	@include rem-fallback(padding-right, $select-padding-horizontal);
-  	@include rem-fallback(padding-left, $select-padding-horizontal);
+    @include rem-fallback(padding-left, $select-padding-horizontal);
+
+    &:last-child { @include border-bottom-radius( $select-input-border-radius ); }
```
Comment (ppathan) [inline src/components/shared/forms/styles/select.scss]: indentation seems off here and on line 355

File: src/styles/_themes.scss [hunk 2]
```diff
@@ -175,35 +181,35 @@ $themes: (
 
     colorBtnText: #212121,
     colorBtnBackground: transparent,
-    colorBtnSvgFill: $colorSmoke,
+    colorBtnSvgFill: #666,
 
-    colorBtnTextHover: $colorWhite,
-    colorBtnBackgroundHover: #dadada,
-    colorBtnSvgFillHover: $colorWhite,
+    colorBtnTextHover: #212121,
```
Comment (avanikp) [inline src/styles/_themes.scss]: Why not define these colors (#212121, #666) in src/styles/_variables.scss and use the variables here ?

Comment (spryor) [reply src/styles/_themes.scss]: That is the plan. We just aren't doing it yet. As part of the broader central alignment we will be sharing sass variables files which can be used here though.

**Pull request merged**